.github/workflows/build_application_images.yml

```diff
@@ -136,10 +134,7 @@ jobs:
           provenance: false
       - name: Check for vulnerability scan findings
         run: |
-          ./scripts/check_ecr_findings "easi-backend" "$GIT_HASH" "7"
-        # We've had too many issues with undefined or low-level vulnerabilities blocking our pipeline
-        # We're going to ignore them for now, but continue to scan and log them.
-        continue-on-error: true
```

Comment:
@azeez-salu I think we need to keep this around (the line and comment above it) while we still have that finding we haven't resolved yet